Automattic/Gravatar-SDK-iOS
# Select avatar on Avatar Picker

### Description

This PR implements selecting a different avatar on the Avatar Picker.

Bigest changes:
- `AvatarModelList` to manage the models array. This is helpful to simplify updating elements in the array.
- `AvatarPickerProfileView` loads the avatar image directly from a URL.
  - In this way we can pass a recently selected avatar url without depending on backend cache times. It also should load the image from already cached images, since the URL is the same than images from the grid. 

<img src="https://github.com/user-attachments/assets/6e2ea5a4-b53e-4733-a4b7-e03cd3634822" width="40%">

### Testing Steps

- Select different avatars from the Avatar Picker.
- Select a recently uploaded avatar.
- Check that the selected avatar gets selected on web (going to the profile url).
